pkg/vault/azure/README.md

```diff
@@ -215,33 +285,137 @@ Example output:
 ]
 ```
 
-## Backend configuration
 
-### Configuration parameters
 
-Name | Type | Required | Description
------|------|:--------:|------------
-vault | URL | ✅ | Vault URL
-tenant_id | UUID | ✅ | Tenant ID
-client_id | UUID | ✅ | Service Principal (application) ID from the registration stage
-client_secret | string | | Used for secret based authentication. Not covered here. Not recommended.
-client_pkcs12_certificate | string | | Path to PKCS #12 file
-client_certificate_thumbprint | string | | Hex or Base64 encoded client certificate hash. Use along with `client_private_key` as an alternative to PKCS #12 flow
-client_private_key | string | | Path to the client private key. Use along with `client_certificate_thumbprint` as an alternative to PKCS #12 flow
-subscription_id | UUID | | Subscription ID. Optional. Only if `Microsoft.ResourceHealth` is enabled (see above)
-resource_group | string | | Resource group name. Optional. Only if `Microsoft.ResourceHealth` is enabled (see above)
+## 
+**Backend configuration**
+
+
+### 
+**Configuration parameters**
+
+
+<table>
```

Comment:
@AbineshECAD does this qualify as markdown? Will this look good on the website? I don't know if this is what we want in our readme

Reply:
This is for making it bold. I will check once whether it's a must and remove it if not needed.